_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC. The proposal is designed before the implementation of *major
```

Comment:
`during GSoC` instead of `during the GSoC` would have to be used. Using "the" was correct with "GSoC duration"  but with only "GSoC" the world "the" needs to be removed.